RELEASE.md

```diff
@@ -15,6 +15,8 @@ Please follow the established format:
 
 - Fix for dataset existence check in factory pattern discovery (#1659)
 - Remove support for Python 3.7 (#1660)
+- Adjust CLI to use 'kedro viz run' instead of 'kedro run' (#1671)
+- Add 'viz' as new CLI command group (#1671)
```

Comment:
Going to go with just the 1 bullet point for now if suitable ("Adjust CLI to use 'kedro viz run' instead of 'kedro run'"), had a chat quickly w/ Rashida and decided that the second change isn't needed to be displayed to users for the time being